hack/test-cmd.sh

```diff
@@ -147,6 +147,18 @@ openshift start \
   --etcd-dir="${ETCD_DATA_DIR}" \
   --images="${USE_IMAGES}"
 
+# validate config that was generated
+[ "$(oadm validate master-config ${MASTER_CONFIG_DIR}/master-config.yaml 2>&1 | grep SUCCESS)" ]
```

Comment:
our CLI can't make up its mind whether the verb or the noun is the grouper... we have a lot of instances of both